### PR TITLE
Makefile: minimum dependencies for dev environment

### PR DESCRIPTION
**Pull request by marwan-at-work**
Fixes dev part of https://github.com/gomods/athens/issues/262

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,15 +35,24 @@ olympus-docker:
 	docker build -t gopackages/olympus -f cmd/olympus/Dockerfile .
 
 .PHONY: dev
+
```
Comment (arschles) [inline Makefile]: would you mind adding the `.PHONY` here? same with the other two targets below

**Comment by marwan-at-work** [comment]
@arschles done!

File: Makefile [hunk 2]
```diff
@@ -34,16 +34,27 @@ test-unit:
...
 	docker-compose -p athensdev up -d mongo
 	docker-compose -p athensdev up -d redis
-	docker-compose -p athensdev up -d minio
-	docker-compose -p athensdev up -d jaeger
-	echo "sleeping for a bit to wait for the DB to come up"
-	sleep 5
+
+.PHONY: down
+down:
+	docker-compose -p athensdev down
```
Comment (michalpristas) [inline Makefile]: will this tear down `make alldeps`?
maybe we should merge athensdev and athensdep
or create another down for deps

**Comment by marwan-at-work** [comment]
@michalpristas good point, merged the project names.

**Pull request merged**